### DISCUSSION TITLE
Feature request: add templates and test subcommands for acceptance tests

### DISCUSSION THREAD

**Discussion by EdgeJ** [puppetlabs/pdk#1014] [Feature Request]
PDK is handy for developing modules and doing unit tests with puppet-rspec.  However, there is not currently a way to easily/quickly/repeatedly set up acceptance tests with the Beaker test framework, using `beaker-rspec`.  It would be nice to have PDK set up boilerplate `spec/spec_helper_acceptance.rb` and `spec/acceptance/nodesets/default.yml` files (the latter, possibly with information from `metadata.json` to figure out what nodes to add to the `default.yml` file), as well as the relevant gems into Gemfile.  Boilerplate specs themselves might be trickier, but it certainly would be a 'nice-to-have'.  

This would help folks quickly get started with acceptance testing as well as help make it easier to understand what some sane baseline settings are for running beaker.

**Comment by scotje** [comment]
Acceptance testing helpers (based on puppetlabs/litmus) need an RFC to describe the new commands and workflow, hopefully that is something we will be writing soon. (Although we always welcome a community written RFC!)
